yapapi/config.py

```diff
@@ -48,6 +48,12 @@ class ApiConfig:
     net_url: Optional[str] = field(default_factory=partial(os.getenv, "YAGNA_NET_URL"))
     activity_url: Optional[str] = field(default_factory=partial(os.getenv, "YAGNA_ACTIVITY_URL"))
 
+    repository_url: Optional[str] = field(
```

Comment:
this is not part of yagna API and shouldn't be part of yagna low level API's config